articles/javascript-iterations-which-one-is-fast/index.md

```diff
@@ -0,0 +1,657 @@
+Sometimes you want to repeat a task several times. Let's say to log "hello world" six times. Here is a simple JavaScript code block do that:
+```js
+console.log("hello world")
+console.log("hello world")
+console.log("hello world")
+console.log("hello world")
+console.log("hello world")
+console.log("hello world")
+```
+The code above will correctly repeat the assigned tasks six times. However, as the number of the task grows, say 100 hello worlds, the code will get messy affecting code readability. It is hard to write the same line of code repeatedly 100 times. Looping through these tasks will affect execution performance drastically. This is where JavaScript loops come in rescue. JavaScript can control the above loop more cleanly with a few lines of codes.
+
+***For example:***
+```js
+for (let loop = 0; loop < 6; loop++) {
+  console.log("hello world")
+}
+```
+There we have it, two lines of code give us a perfect six hello world as we anticipated. Shorter code means faster development time, code readability, and lesser performance overheads.
+
+JavaScript has different kinds of iterations statements called loops. They include `for`, `while`, `do while`, `for in`, `for of`, and `for each`. They all do the same job, to repeat an action several times, but have different ways to start and end a loop.ster? When it comes to loops, the most significant questions are which loop to use? Which loop is fastest?
+
+In this guide, we will discuss these loops with simple and basic examples, compare and test the execution performance of each loop.
+
+### For Loop
+For loop is the commonly used loop. If you are not new to the world of programming, you might have come across a for loop statement under other programming languages such as `java`, `c++`, `c`, etc. A for statement loops through a specified or fixed number of elements until the specified condition is met.
+
+For statement consist of the following parts:
+```js
+for (initializer; condition; increment){
+    statement //loop body
+}
+```
+- Initialization expression - it is a variable declaration that initializes a loop counter `loop = 0`.
```

Comment:
Could you please explain the purpose of `loop = 0` in the statement? It is unclear at this point and it can be confusing